search-commons/src/main/java/no/unit/nva/search/IndexingClient.java

```diff
@@ -67,31 +66,27 @@ public IndexingClient(RestHighLevelClientWrapper elasticSearchClient) {
         logger.info(INITIAL_LOG_MESSAGE, ELASTICSEARCH_ENDPOINT_ADDRESS, ELASTICSEARCH_ENDPOINT_INDEX);
     }
 
-    public void addDocumentToIndex(IndexDocument indexDocument) throws SearchException {
-        try {
-            doUpsert(indexDocument.toIndexRequest());
-        } catch (Exception e) {
-            throw new SearchException(e.getMessage(), e);
-        }
+    public Void addDocumentToIndex(IndexDocument indexDocument) throws IOException {
+        elasticSearchClient.index(indexDocument.toIndexRequest(), RequestOptions.DEFAULT);
```

Comment:
I have inlined the code of doUpsert because it is paradoxical to throw a Search Exception when we do indexing.